wpilibj/src/main/java/edu/wpi/first/wpilibj/ADIS16448_IMU.java

```diff
@@ -108,9 +108,9 @@ public enum CalibrationTime {
     _32s(10),
     _64s(11);
 
-    private int value;
+    private final int value;
 
-    private CalibrationTime(int value) {
+    CalibrationTime(int value) {
```

Comment:
Why was this changed from private to package-private?

Reply:
enum constructors are implicitly private